README.md

```diff
@@ -4,5 +4,17 @@
 
 This application requests the following additional permissions:
 
-- Access to the host filesystem: `--filesystem=host`.
-  This is currently required for drag & drop support.
+- Read/write access to the host filesystem: `--filesystem=host`.
+  Required for ‘Open Recent’ and for drag & drop support,
+  see [QTBUG-91357](https://bugreports.qt.io/browse/QTBUG-91357).
+  Write access required for ‘Annotations’ and ‘Digitally Sign…’.
+- Access to network: `--share=network`.
+  Required for opening URLs.
```

Comment:
Do you mean that Okular would open remote files directly? Or is it opening URLs from say PDFs in Okular?

Reply:
Ah, never mind, it's already included so that's OK.